services/rollout-service/pkg/cmd/server.go

```diff
@@ -0,0 +1,167 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/argoproj/argo-cd/v2/pkg/apiclient"
+	argoio "github.com/argoproj/argo-cd/v2/util/io"
+
+	"github.com/freiheit-com/kuberpult/pkg/api"
+	"github.com/freiheit-com/kuberpult/pkg/logger"
+	"github.com/freiheit-com/kuberpult/pkg/setup"
+	"github.com/freiheit-com/kuberpult/services/rollout-service/pkg/service"
+	"github.com/freiheit-com/kuberpult/services/rollout-service/pkg/versions"
+	"github.com/kelseyhightower/envconfig"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+	"google.golang.org/protobuf/types/known/emptypb"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	grpctrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/grpc"
+)
+
+type Config struct {
+	CdServer      string `default:"kuberpult-cd-service:8443"`
```

Comment:
I would prefer if we parse variable without envconfig and use `os.Lookup` instead, see here: https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/WORKSPACE_DOCUMENT/-MnQK-Om1MZM1qBy6TV9/

(but this could be done in another PR)

Reply:
We are doing this in many places already and the recommendation is just against default values, not against using envparse. I would post-pone this.